### ISSUE TITLE
Recording of incidental by-catches  - percentage of observation

### ISSUE THREAD

**Issue by luciazarauz** [ices-tools-dev/RDBES#74]
Dear all,

I remember that some time ago, it was discussed in different fora the need to include the percentage of the actual hauling process that was properly checked for by-catch in order to distinguish non-observations ( = missing values) from zero-observation (true 0s).
However, I cannot find this variable in the Data Model (nor in tables nor in the documentation)

This is probably a very silly question, but I woudl appreciate any help with it :-) 

Thanks!

**Comment by nmprista** [comment]
You can record percentage of haul observed for by-catch by including a visual row in SS table associated with a by-catch list. Then in the SA table you can use SAnumberTotal and SAnumber to describe the percentage you observed. Take a look at Annex IV in documentation for some examples.

**Comment by luciazarauz** [comment]
Thanks Nuno. 
Table SA is not in the examples of Annex IV, and I was getting confused. 
In this logic, the number of bycatch individuals observed, would it be in a lower hierarchy table for each row in SA table?

**Comment by nmprista** [comment]
Yes, I believe so. I think you could either a) report it as hierarchy "C" assuming you have some parameter associated to them; or b) if you don't have bio parameters to it (i.e., you just have a number or a weight), then as a SAsubsample of the SAparent detailing the visual observation. Let know if this does not work and attach an example [data model is not yet fully tested for by-catch issues and every example is precious]

**Comment by luciazarauz** [comment]
OK, thanks a lot. 
At the moment we are not ready to give a proper example. We are just preparing our BD to host PETs data (and we wanted to understand RDBES format so that we can take into account future needs).  
but if you need to test by-catch issues, just ask and we can do something quick to test :-)

**Comment by nmprista** [comment]
Thanks. One note I need to add (not closing the issue yet on account of that). Reporting e.g., total minutes recorded from a fishing operation and minutes observed (e.g., using a camara) may not be the right values to report as SAnumberTotal and SAnumberSampled, respectively. The situation is looking to me analogous to the Kg/kg situation described in Annex 10 of WKRDB-EST report. I will put this up for discussion at WKRDB-EST2 tomorrow and confirm if we need to improve the guidelines on that before I closing the issue.

**Comment by nmprista** [comment]
analysis done at WKRDB-EST2
[WKRDBEST2_Annex5_Issue6_PercCoverageByCatch.docx](https://github.com/ices-tools-dev/RDBES/files/6047591/WKRDBEST2_Annex5_Issue6_PercCoverageByCatch.docx)

**Comment by nmprista** [comment]
Discussion at estimation subgroup
- This is connected with #117 #71
- does not appear correct to declare chunks of minutes in numSampled and numTotal. If numSampled represents sampling units then declaring chuncks of minutes there is not correct.
- discussions held in other fora indicated that minutes like weight are auxiliary variable used in ratios. As it could make sense some sort of auxiliary variables xxxSampled, xxxTotal and xxxType that could be used for both minutes and weights (note that right now only weights can be declared) and from which % could be calculated.
- It is also an option to ask submitters to declare numSampled ==  1 and leave numTotal empty when numTotal is to be estimated [this is particularly important when sampling fraction is large and integers do not approximate well numTotal] and instead fill in the estimated inclusion probability.

The last two bullet points combined appeared to subgroup to be the best solution. The last bullet can be a good guideline in the short term if needed. Issue needs further discussion in core-group.

**Comment by luciazarauz** [comment]
thanks for the clarification!
